mep/context_processors.py

```diff
@@ -6,6 +6,7 @@ def template_settings(request):
 
     context_extras = {
         'SHOW_TEST_WARNING': getattr(settings, 'SHOW_TEST_WARNING', False),
-        'INCLUDE_ANALYTICS': getattr(settings, 'INCLUDE_ANALYTICS', False)
+        'GTAGS_ANALYTICS_ID': getattr(settings, 'GTAGS_ANALYTICS_ID', False),
+        'GTAGS_ANALYTICS_ENV': getattr(settings, 'GTAGS_ANALYTICS_ENV', False)
```

Comment:
Would make more sense to use `None` than `False` as the default for these, since they aren't boolean flags. (But not a big deal.)

Reply:
makes sense; I just copied from the `SHOW_TEST_WARNING` one. i'll change it